actinia_stac_plugin/core/stac_redis_interface.py

```diff
@@ -0,0 +1,188 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Copyright (c) 2018-2021 mundialis GmbH & Co. KG
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+Process Chain Template Management
```

Comment:
In this module are a lot of left-overs from the actinia-templates in comments and variable names. On the one hand it would be nice to get rid of them and use more fitting names, on the other hand it would be better to find a solution maybe within actinia_core to reuse this class without copying it. (No action required for now but something we should keep in mind)

Reply:
Or maybe add a comment, that it is not adjusted by purpose but will be taken care of at some point.